### PR TITLE
docs: add links to TinyCore ISOs and kernel sources

### PR DESCRIPTION
**Pull request by orangecms**

### CODE REVIEW

**Comment by bjakobson** [comment]
looks good, you do a great job with detailed descriptions :)

**Comment by orangecms** [comment]
> looks good, you do a great job with detailed descriptions :)

Thank you, that is very assuring! :)

File: README.md [hunk 1]
```diff
@@ -5,12 +5,21 @@ over the web.
 
 ## Concept
 
-The key idea is to have the respective distro's image copied to RAM and preserve
-it throughout a kexec. That is achieved by using a pmem device, which is created
+The webboot bootloader works as follows:
+
+1. fetch an OS distro release ISO from the web
+2. copy the ISO to memory
```
Comment (rminnich) [inline README.md]: memory -> persistent memory, e.g. /dev/pmem0

Comment (orangecms) [reply README.md]: I wanted to keep that detail for the full explanation below, so it doesn't confuse people early who don't know that concept

**Pull request merged**